src/renderers/testing/ReactShallowRenderer.js

```diff
@@ -12,78 +12,37 @@
 
 'use strict';
 
+const checkPropTypes = require('prop-types/checkPropTypes');
 const React = require('react');
-const ReactTestRenderer = require('react-test-renderer');
 
-const emptyFunction = require('fbjs/lib/emptyFunction');
+const emptyObject = require('fbjs/lib/emptyObject');
 const invariant = require('fbjs/lib/invariant');
 
-const ShallowNodeMockComponent = ({children}) =>
-  React.Children.toArray(children);
+const {
+  ReactDebugCurrentFrame,
+} = React.__SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED;
```

Comment:
Shouldn't `const { ReactDebugCurrentFrame } = require('ReactGlobalSharedState')` work? If not we need to fix that.